SingularityUI/app/handlebarsHelpers.coffee

```diff
@@ -103,3 +103,6 @@ Handlebars.registerHelper 'humanizeFileSize', (bytes) ->
 Handlebars.registerHelper 'usernameFromEmail', (email) ->
     return '' if not email
     email.split('@')[0]
+
+Handlebars.logger.level = 0;
+
```

Comment:
what does setting this do? was this something needed for testing, or is it not a big deal to leave here?